File: Talksy-API/src/main/java/com/amouri_dev/talksy/entities/notification/Notification.java
package com.amouri_dev.talksy.entities.notification;

import com.amouri_dev.talksy.entities.message.MessageType;
import lombok.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class Notification {

    private Long chatId;
    private String content;
    private Long senderId;
    private Long recipientId;
    private String chatName;
    private MessageType messageType;
    private NotificationType type;
    private byte[] media;
}

File: talksy-ui/src/app/services/models/notification.ts
export interface notification {
  chatId?: number,
  message?: string,
  senderId?: number,
  recipientId?: number,
  messageType: 'TEXT' | 'IMAGE' | 'VIDEO' | 'AUDIO';
  type?: 'SEEN' | 'MESSAGE' | 'IMAGE' | 'AUDIO' | 'VIDEO';
  chatName?: string,
  media?: string;
}